packages/node/src/utils/project.ts

```diff
@@ -0,0 +1,18 @@
+import fs from 'fs';
+import os from 'os';
+import path from 'path';
+import tar from 'tar';
+
+export async function processPath(projectPath: string): Promise<string> {
+  const stats = fs.statSync(projectPath);
+  if (stats.isFile()) {
+    const sep = path.basename(projectPath);
```

Comment:
`sep` means seperator, we don't really need the original basename to name temp dir

Reply:
Fixed